micro-async-data-writer/src/main/java/com/aol/micro/server/async/data/cleaner/ConfigureDataWriter.java

```diff
@@ -0,0 +1,55 @@
+package com.aol.micro.server.async.data.cleaner;
+
+import java.util.List;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.aol.cyclops.data.collections.extensions.standard.ListX;
+import com.aol.micro.server.async.data.writer.AsyncDataWriter;
+import com.aol.micro.server.async.data.writer.MultiDataWriter;
+import com.aol.micro.server.manifest.ManifestComparator;
+import com.google.common.eventbus.EventBus;
+
+@Configuration
+public class ConfigureDataWriter {
+
+	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+	@Autowired(required=false)
+	private List<ManifestComparator> defaultComparators = ListX.empty();
+	@Value("${asyc.data.writer.threads:1}")
+	private int writerThreads=1;
+	@Value("${asyc.data.writer.multi:false}")
+	private boolean multiWriterOn=false;
+	
+	@Autowired
+	private EventBus bus;
+	
+	@Bean
+	public AsyncDataWriter<?> defaultDataWriter(){
+		if(defaultComparators.size()>0){
+			System.err.println("Warning :: multiple ManifestComparators configured as Spring bean, using the first configured bean for the Default AsyncDataWriter, recommended approach is to configure your own DataWriters as needed.");
```

Comment:
This one isn't debug, if the users logger isn't configured correctly the warning won't be seen. 99% of the time you won't have more than one ManifestComparator bean, and I think a warning on startup to System.err when you do is minor cost, at least it will help make users aware that there are multiple ManifestComparator beans created.